packages/core/src/lib/markdown-it/markdown-it-icons.js

```diff
@@ -7,6 +7,13 @@ module.exports = require('markdown-it-regexp')(
         let iconFontName = match[2];
         let iconClass = match[3];
 
+        // ensure octicons exist
+        if (iconFontType === 'octicon' || iconFontType === 'octiconlight') {
+            if (!octicons.hasOwnProperty(iconFontName)) {
+              return `<span aria-hidden="true"></span>`;
+            }
+        }
```

Comment:
is it possible to combine the below checks (`iconFontType === 'octicon'` and `iconFontType === 'octiconlight')  and only adjust the style conditionally?
so we have less repeated code down there, and integrate this check only where its needed

Reply:
Right, that would be much better, I'll do that